rfcs/20190308-addons-proxy-maintainership.md

```diff
@@ -0,0 +1,174 @@
+account during the periodic review of their subpackage/submodule.
+
+### Retiring from Proxy Maintainership
+
+If a proxy maintainer decides that they no longer wish to maintain one 
+or more of their subpackages, they **must** commit to the following procedure:
+
+* Send an email to addons@tensorflow.org and list all code that they are 
+no longer able to maintain. It is usually a good idea to shortly describe the 
+state of code, e.g. whether the packages have open bugs, whether they are difficult 
+to maintain, etc.
+
+* The proxy maintainer must remove themselves from the `CODEOWNERS.md` 
+file, with `<!--maintainer-needed-->` listed in the place of 
+maintainers. This will help other developers to find subpackages with no 
+maintainers.
+
+## Repository Growth and Review 
+### Periodic Review
+
+Given the nature of this repository, subpackages and submodules may become less 
+and less useful to the community as time goes on. In order to keep the 
+repository sustainable, we'll be performing bi-annual reviews of our code to 
+ensure everything still belongs within the repo. Contributing factors to this 
+review will be:
+
+1. Number of active maintainers
+2. Amount of OSS use
+3. Amount of issues or bugs attributed to the code
+
+Because we encourage TF-Addons to be used in production, we'll plan on a 
+graceful deprecation process, instead of a swift removal.
+
+### Adding a New Subpackage
+
+The SIG Addons core team will review the submitted files 
+(for details on coding style, requirements, and testing, please refer 
+to [`CONTRIBUTING.md`](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md)) 
+and help bring the contribution to an acceptable quality. Once the 
+package is ready, it will be merged and the relevant bugs will be 
+closed (if there are any).
+
+Please note that the Addons team reserves the right to reject new 
+subpackages, especially if they would otherwise qualify for removal per 
+its quality standards. Example reasons for rejection include:
+
+* Having known major bugs or security issues.
+* Duplicating functionality in the TensorFlow core API.
+* Having deprecated upstream dependencies.
+* Failling to obtain an appropriate code review.
+* Having no general usefulness for TensorFlow users.
+
+### Moving a Subpackage from tf.contrib
+
+If a potential maintainer wishes to migrate a subpackage that was deprecated 
+as part of `tf.contrib`, they must explicitly note that on their 
+submission. A potential maintainer would be required to:
+
+* Demonstrate that the code is useful to the community at large.
+* Modify the code to utilize TF 2.x functionality
+
+**Useful Links:**
+* [SIG Addons RFC](https://github.com/tensorflow/community/blob/ef626896f30130dfc3b5e75126c94624b689a943/rfcs/20181214-move-to-addons.md).
+* [SIG Addons Charter](https://github.com/tensorflow/community/blob/master/sigs/addons/CHARTER.md).
+
+## Questions and Discussion Topics
```

Comment:
How are the docs surfaced? (if at all?)
Seems like guides/tutorials/narrative should not surface on tensorflow.org. What about API docs?

Reply:
initial thoughts, if I'm not wrong, we prefer to host it on tensorflow.org, please see the discussion in https://github.com/tensorflow/addons/issues/46

Reply:
Great comment! As discussed last week - API documentation for `addons` should conform to the existing documentation standards for TensorFlow, and ideally be centralized with core, `tfp`, `tfds`, `tfdv`, etc.